packages/apollo-server-lambda/package.json

```diff
@@ -26,7 +26,6 @@
   },
   "dependencies": {
     "@apollographql/graphql-playground-html": "1.6.24",
-    "@types/aws-lambda": "^8.10.31",
```

Comment:
The `APIGatewayProxyEvent` type provided by this package is used within `apollo-server-express`:

https://github.com/apollographql/apollo-server/blob/84e1aa5ad3ee2acf410ca8984cfc68e90d4f8a66/packages/apollo-server-lambda/src/ApolloServer.ts#L1-L5


...and re-exported on the `CreateHandlerOptions` interface on the `onHealthCheck` property:

https://github.com/apollographql/apollo-server/blob/84e1aa5ad3ee2acf410ca8984cfc68e90d4f8a66/packages/apollo-server-lambda/src/ApolloServer.ts#L15-L25